test/feature/StrongMode/Ellision.js

```diff
@@ -0,0 +1,15 @@
+// Options: --strong-mode
+
+'use strong'
+
+let a = [0, 1, 2, ];
+assert.equal(3, a.length);
+
+let [, b, , c] = [0, 1, 2, 3];
+assert.equal(1, b);
+assert.equal(3, c);
+
+// Not working ATM.
+// let d;
+// [, d] = [0, 1];
```

Comment:
I haven't seen this pattern in any ES6 code yet. I know it happens a bit in FP where you only want the tail of a list and don't care about the head.

I agree that postponing this is the right thing to do at the moment.